mflike/mflike.py

```diff
@@ -379,18 +390,50 @@ def get_foreground_model(fg_params, fg_model,
         {"ell": ell, "ell_0": ell_0})
     model["tt", "cibp"] = fg_params["a_p"] * cibp(
         {"nu": frequencies, "nu_0": nu_0,
-         "temp": fg_params["T_d"], "beta": fg_params["beta_p"]},
-        {"ell": ell, "ell_0": ell_0, "alpha": 2})
+        "temp": fg_params["T_d"], "beta": fg_params["beta_c"]},
+        {"ell": ell_clp, "ell_0": ell_0clp, "alpha": 1})
     model["tt", "radio"] = fg_params["a_s"] * radio(
-        {"nu": frequencies, "nu_0": nu_0, "beta": -0.5 - 2},
-        {"ell": ell, "ell_0": ell_0, "alpha": 2})
+        {"nu": frequencies, "nu_0": nu_0, "beta": -0.5 - 2.},
+        {"ell": ell_clp, "ell_0": ell_0clp,"alpha":1})
     model["tt", "tSZ"] = fg_params["a_tSZ"] * tsz(
         {"nu": frequencies, "nu_0": nu_0},
         {"ell": ell, "ell_0": ell_0})
     model["tt", "cibc"] = fg_params["a_c"] * cibc(
+        {"nu": fdustw, "nu_0": nu_0,
+        "temp": fg_params["T_d"], "beta": fg_params["beta_c"]},
+        {'ell':ell, 'ell_0':ell_0})
+    model["tt", "dust"] = fg_params["a_g"] * dust(
+        {"nu": frequencies, "nu_0": nu_0,
+        "temp": 19.6, "beta": 1.5},
+        {"ell": ell, "ell_0": 500., "alpha": -0.6})
+    model["tt","tSZ_and_CIB"] = tSZ_and_CIB(
+        {'kwseq': (
+        {'nu':frequencies, 'nu_0': nu_0},
+        {'nu': frequencies, 'nu_0': nu_0, 'temp': fg_params['T_d'], 'beta': fg_params["beta_c"]} 
+                            )},
+        {'kwseq': ( 
+        {'ell':ell, 'ell_0': ell_0, 
+        'amp': fg_params['a_tSZ']},
+        {'ell':ell, 'ell_0': ell_0, 'amp': fg_params['a_c']},
+        {'ell':ell, 'ell_0': ell_0, 
+        'amp': - fg_params['xi'] * np.sqrt(fg_params['a_tSZ'] * fg_params['a_c'])}
+                )})
+
+    model["ee", "radio"] = fg_params["a_psee"] * radio(
+        {"nu": frequencies, "nu_0": nu_0, "beta": -0.5 - 2.},
+        {"ell": ell_clp, "ell_0": ell_0clp,"alpha":1})    
+    model["ee", "dust", "wide"] = fg_params["a_gee"] * dust(
```

Comment:
I guess `wide` refers to `deep/wide` fields for ACT. I think it will not work later when we loop over the foreground components since the code will look for ["ee", "dust"]. I think this also a relic since there is not `deep/wide` in the TE dust component